pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -291,6 +293,17 @@ func doCreateBackupSchedules(
 		}
 	}
 
+	_, updatesErrorMetrics := scheduleOptions[optUpdatesErrorMetrics]
+
+	if updatesErrorMetrics {
```

Comment:
what's the motivation behind adding a backup schedule option for this change?

Reply:
this feature is something that we want to be opt-in so that e.g. customer created schedules don't write this metric by default. This is in line with the similar RPO metric that's controlled by `optUpdatesLastBackupMetric`

Reply:
I see, that clarifies things. thanks! It may be worth adding one sentence explaining the opt-in motivation in the commit message or in the code, namely: " customer created schedules don't write this metric by default."